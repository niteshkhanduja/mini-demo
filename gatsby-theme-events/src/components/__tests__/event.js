import React from "react"
import renderer from "react-test-renderer"

import Evet from "../event"

describe("Evet", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Evet siteTitle="Default Starter" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})