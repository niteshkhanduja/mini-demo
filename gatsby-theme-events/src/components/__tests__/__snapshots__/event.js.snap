// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Evet renders correctly 1`] = `
<div>
  <h1
    css={
      Object {
        "map": undefined,
        "name": "1ofhypn",
        "next": undefined,
        "styles": "
  text-decoration: underline;
  ",
        "toString": [Function],
      }
    }
  >
    (
    )
  </h1>
  <p
    css={
      Object {
        "map": undefined,
        "name": "1yov2b9",
        "next": undefined,
        "styles": "
color:red

",
        "toString": [Function],
      }
    }
  >
    Website :
    <a />
  </p>
  <p>
    Starting date:
  </p>
  <p>
    Ending Date :
  </p>
</div>
`;
